distributed/scheduler.py

```diff
@@ -2685,7 +2740,7 @@ def start_ipython(self, stream=None):
 
 
 def decide_worker(dependencies, stacks, processing, who_has, has_what, restrictions,
-                  loose_restrictions, nbytes, key):
+                  loose, nbytes, key):
```

Comment:
This seems like a good move.  I would expect a lot of `test_scheduler.py` tests to get angry with you though :)